src/falkordb.ts

```diff
@@ -103,6 +103,12 @@ export default class FalkorDB extends EventEmitter {
     static async connect(options?: FalkorDBOptions) {
         const redisOption = (options ?? {}) as RedisClientOptions<{ falkordb: typeof commands }, RedisFunctions, RedisScripts>;
 
+        // If the URL is provided, and the protocl is falkordb replace it with redis for the underline redis client
+        // e.g. falkordb://localhost:6379 -> redis://localhost:6379
+        if (redisOption.url && redisOption.url.startsWith('falkordb')) {
```

Comment:
take a look at https://github.com/FalkorDB/falkordb-py/blob/6613929b980dbac426f98a7e9bc2e7538d9e80ec/falkordb/falkordb.py#L125 it is falkor not falkordb

Reply:
so falkor:// and not falkordb://?